iocBoot/templates/st.cmd.template

```diff
@@ -191,7 +198,7 @@ cd "$(IOC_TOP)"
 ## PLC Project Database files ##
 {% if plc.tmc %}
 {% for db in db_filenames %}
-dbLoadRecords("{{ db }}", "PORT={{ asyn_port }},{{ db_parameters }}")
+dbLoadRecords("{{ db }}", "PORT=$(ASYN_PORT},{{ db_parameters }}")
```

Comment:
Should that be
`PORT=$(ASYN_PORT)`

Reply:
I think you might be looking at a stale branch somehow (refresh on your browser?). We fixed that one yesterday, and it was the biggest initial bug that caused none of the records to load :)

https://github.com/pcdshub/ads-ioc/pull/86/files#diff-1f8de0f9489ee555f92412e0c794d381b0894fd8ac196290f7926da72550e334R203

Reply:
Indeed, I was looking at a specific commit just before that fix! Sorry about that

Reply:
I've done that many-a-time - no problem!